packages/react-server/src/ReactFlightServer.js

```diff
@@ -3986,7 +4002,7 @@ export function stopFlowing(request: Request): void {
 // This is called to early terminate a request. It creates an error at all pending tasks.
 export function abort(request: Request, reason: mixed): void {
   try {
-    if (request.status === OPEN) {
+    if (request.status === PENDING) {
```

Comment:
This doesn't seem right. PENDING is a Task status not a request status. (Both happens to currently be zero.)